projects/packages/woocommerce-analytics/src/class-checkout-flow.php

```diff
@@ -154,7 +154,6 @@ public function capture_checkout_view() {
 			return;
 		}
 
-		$is_in_checkout_page                       = $checkout_page_id === $post->ID ? 'Yes' : 'No';
```

Comment:
This is already determined by the conditional above

Reply:
It's a good point, but maybe we should still return early in that method when `$post` returns null?

Reply:
Good call! eee295d